### PR TITLE
install yarn

### PR DESCRIPTION
**Pull request by vincentfretin**
Install yarn package manager for ubuntu/debian and mac.

### CODE REVIEW

File: fabfile.py [hunk 1]
```diff
@@ -1544,6 +1545,19 @@ def install_java():
 
 
 @task
+def install_yarn():
+    """Install yarn"""
+    if not env.mac:
+        if not exists('/etc/apt/sources.list.d/yarn.list'):
```
Comment (ayyazdaniaryan) [inline fabfile.py]: All looks well. One question. If one of our production servers are on Ubuntu 14.04, will this step also be required? 
https://nodejs.org/en/download/package-manager/#debian-and-ubuntu-based-linux-distributions

Comment (vincentfretin) [inline fabfile.py]: you already install an updated node version via nodeenv if you don't have a recent node version on the system, see update_node task in fabfile.py

Comment (vincentfretin) [inline fabfile.py]: on my machine I installed node from nodesource repository yeah.

**Comment by swergas** [comment]
Looks good to me too

**Pull request merged**